clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -317,7 +317,9 @@
             NoReassignmentInProgressException::new),
     GROUP_SUBSCRIBED_TO_TOPIC(86, "Deleting offsets of a topic is forbidden while the consumer group is actively subscribed to it.",
         GroupSubscribedToTopicException::new),
-    INVALID_RECORD(87, "This record has failed the validation on broker and hence be rejected.", InvalidRecordException::new);
+    INVALID_RECORD(87, "This record has failed the validation on broker and hence be rejected.", InvalidRecordException::new),
+    PURGE_IN_QUEUE(88, "This record has failed because other records from the same batch has been rejected on " +
```

Comment:
I think we decided not to do this for now, right?

Reply:
My bad, will remove.